src/controllers/profileController.ts

```diff
@@ -57,26 +57,28 @@ export default class ProfileController {
             workEmail
           );
           return errorResponseOrSuccessResponse.map(response => {
-            const emailText = localeIt.profileController.editProfile.notificationEmail.content.replace(
-              "%s",
-              user.givenName
-            );
-            // tslint:disable-next-line:no-floating-promises
-            this.emailService
-              .send({
-                html: emailText,
-                subject:
-                  localeIt.profileController.editProfile.notificationEmail
-                    .subject,
-                text: emailText,
-                to: workEmail
-              })
-              .catch(error =>
-                log.error(
-                  "Failed to send email notification for work email change. %s",
-                  error
-                )
+            if (response.value.work_email !== response.value.email) {
+              const emailText = localeIt.profileController.editProfile.notificationEmail.content.replace(
+                "%s",
+                user.givenName
               );
+              // tslint:disable-next-line:no-floating-promises
```

Comment:
please add a comment with a "todo" linked to a pivotal story to refactor this kind of task(s) with some external worker processes linked to a queue and the reason why the promise is floating atm